### PR TITLE
distinguish between local and prod secrets

### PR DESCRIPTION
**Pull request by iainhouston**

### CODE REVIEW

File: provisioning/tasks/config.yml [hunk 1]
```diff
@@ -12,5 +12,5 @@
   with_fileglob:
     - "{{ config_dir }}/config.yml"
     - "{{ config_dir }}/local.config.yml"
-    - "{{ config_dir }}/secrets.yml"
+    - "{{ config_dir }}/{{ lookup('env', 'DRUPALVM_ENV')|default(drupalvm_env, true)|default(ansible_env.DRUPALVM_ENV)|default(omit) }}.secrets.yml"
```
Comment (oxyc) [inline provisioning/tasks/config.yml]: Could you keep the old behavior as well as we talked about in https://github.com/geerlingguy/drupal-vm/issues/1522#issuecomment-333314535

Comment (oxyc) [reply provisioning/tasks/config.yml]: True, that sounds like a good solution but I do see a use case for separate `<env>.secrets.yml` files, eg separate staging/prod environments. But maybe we shouldn't add more features unless someone actually asks for it.

Comment (geerlingguy) [reply provisioning/tasks/config.yml]: It would be nice to have... but if it makes things harder, let's keep the current level of support only.

**Comment by geerlingguy** [comment]
See comments on #1522

**Pull request closed without merging**